### PR TITLE
Updating samples for preview.2

### LINKED ISSUE #54: Update samples repo for preview.2

### PR DESCRIPTION
**Pull request by DamianEdwards**
WIP branch for updating samples to preview.2. Make changes and/or additions here as necessary.

Fixes #54, #48

### CODE REVIEW

File: NuGet.config [hunk 1]
```diff
@@ -2,13 +2,19 @@
 <configuration>
   <packageSources>
     <clear />
+    <add key="dotnet8" value="https://pkgs.dev.azure.com/dnceng/public/_packaging/dotnet8/nuget/v3/index.json" protocolVersion="3" />
```
Comment (DamianEdwards) [inline NuGet.config:5]: This is temporary until the preview.2 packages are released on nuget.org

File: build.cmd [hunk 2]
```diff
@@ -1,6 +1,6 @@
 @ECHO OFF
 
-dotnet workload update
-dotnet workload install aspire
+dotnet workload update --skip-sign-check
+dotnet workload install aspire --skip-sign-check
```
Comment (DamianEdwards) [inline build.cmd:4]: This is temporary until the official workload packages are published.

File: build.sh [hunk 3]
```diff
@@ -1,6 +1,6 @@
 #!/usr/bin/env bash
 
-dotnet workload update
-dotnet workload install aspire
+dotnet workload update --skip-sign-check
+dotnet workload install aspire --skip-sign-check
```
Comment (DamianEdwards) [inline build.sh:5]: This is temporary until the official workload packages are published.

File: samples/AspireWithNode/AspireWithNode.AppHost/Program.cs [hunk 4]
```diff
@@ -1,12 +1,15 @@
...
+var cache = builder.AddRedis("cache");
 
 var weatherapi = builder.AddProject<Projects.AspireWithNode_AspNetCoreApi>("weatherapi");
 
 builder.AddNpmApp("frontend", "../NodeFrontend", "watch")
     .WithReference(weatherapi)
     .WithReference(cache)
-    .WithServiceBinding(scheme: "http", env: "PORT");
+    // This is a workaround for https://github.com/dotnet/aspire/issues/1430
+    // .WithServiceBinding(scheme: "http", env: "PORT")
```
Comment (DamianEdwards) [inline samples/AspireWithNode/AspireWithNode.AppHost/Program.cs]: Are you removing this now the fix has been backported?

Comment (davidfowl) [reply samples/AspireWithNode/AspireWithNode.AppHost/Program.cs]: Did

**Review by joperezr — APPROVED** [review]

**Review by timheuer — APPROVED** [review]

**Pull request merged**